src/api/session.js

```diff
@@ -84,9 +84,20 @@ export class SessionCache {
 const sessionCache = new SessionCache();
 
 session.getStatic = async function getStatic(url) {
-  if (sessionCache.has(url)) return sessionCache.lookup(url);
+  if (sessionCache.has(url)) {
+    return sessionCache.lookup(url);
+  }
+
+  let result = await session.get(url).then((r) => r.data);
+  let redirect = result.location;
+
+  // text requests return a 200 but have a location header
```

Comment:
Just to prevent confusion in the future - they do not have a location header like a 302, they return JSON with a location ie:
`{"location": "https://www.example.com/go_here/"}`
I believe `r.data` is fetching the parsed JSON, and `result.location` is accessing the proper property (as exhibited by it working on the deploy preview), but the comment here should be updated

Reply:
Updated